addons/grafana/README.md

```diff
@@ -0,0 +1,10 @@
+# Custom [Grafana](https://grafana.com/) image
+
+[`start.sh`](start.sh) is the entrypoint, and it runs
+[`import_dashboard.sh`](import_dashboard.sh) in the background then
+starts Grafana. `import_dashboard.sh` POSTs the included dasbboards
+and Prometheus data-source config to Grafana once it starts up.
+
+A few env vars are set in the [Dockerfile](Dockerfile) to configure
+Grafana, and more environment-specific ones are set in the [install
+config](/install/kubernetes/templates/addons/grafana.yaml.tmpl).
```

Comment:
Let's add a section on which dashboards are included with a brief (5 words or less) summary of each one.

Reply:
Done